### ISSUE TITLE
Highliting log outputs

### ISSUE THREAD

**Issue by tamuratak** [James-Yu/LaTeX-Workshop#3078]
We can now set a language id to output channels:

- https://github.com/microsoft/vscode/issues/19561#issuecomment-1032641929

With the feature, we can highlight the outputs with custom language syntax, e.g., `latex-workshop-log.tmLanguage.json`, `latex-workshop-compiler-log.tmLanguage.json`. 

@jlelong I think these files do not fit with jlelong/vscode-latex-basics. We had better create them on LaTeX Workshop's side. They don't have to be comprehensive. Even if they are basic ones, they would be very useful.

If you are not interested, I would like to make them.

**Comment by tamuratak** [comment]
> Is the proposed API already in the Insider version?

Yes. You have to set ` "enabledApiProposals": ["outputChannelLanguage"],` in `package.json`, and download `vscode.proposed.outputChannelLanguage.d.ts`. 

- https://code.visualstudio.com/api/advanced-topics/using-proposed-api
- https://github.com/microsoft/vscode/blob/1ecb5f53a739dd84c66b51e2be1ea7f440141db7/src/vscode-dts/vscode.proposed.outputChannelLanguage.d.ts

**Comment by wzh4464** [comment]
It will be a great help! Thank you, developers.

**Comment by ajinkya-k** [comment]
I would like to second this request!

**Comment by jlelong** [comment]
@James-Yu Can we consider that #4331 was good enough to close this issue for now?

**Comment by James-Yu** [comment]
> [@James-Yu](https://github.com/James-Yu) Can we consider that [#4331](https://github.com/James-Yu/LaTeX-Workshop/pull/4331) was good enough to close this issue for now?

Absolutely!
